R/Common_functions.R

```diff
@@ -143,12 +143,12 @@ commonFG <- list(
             stop(paste("An object with name", group_name, "does not exist"))
         }
     },
-    ls=function(recursive=FALSE, detailed=FALSE, index_type=h5const$H5_INDEX_NAME, order=h5const$H5_ITER_NATIVE, link_access_pl=h5const$H5P_DEFAULT,
-        dataset_access_pl=h5const$H5P_DEFAULT, type_access_pl=h5const$H5P_DEFAULT) {
+    ls=function(recursive=FALSE, detailed=FALSE, index_type=h5const$H5_INDEX_NAME, order=h5const$H5_ITER_NATIVE, link_access_pl=h5const$H5P_DEFAULT$id,
+        dataset_access_pl=h5const$H5P_DEFAULT$id, type_access_pl=h5const$H5P_DEFAULT$id) {
```

Comment:
Well - the reason where a few breaking test cases.
Should be fixed now, ids now only used in function call.